### ISSUE TITLE
Bundle composer vendor dependencies with released tarballs to support reproducible build system packaging

### ISSUE THREAD

**Issue by amessina** [cypht-org/cypht#1122]
## 🚀 Feature
Bundling composer vendor dependencies with released tarballs supports build system (like Koji) packaging that doesn't enable network connections.  This is in support of reproducible builds.

**Comment by marclaporte** [comment]
Related: https://github.com/cypht-org/cypht/issues/597

**Comment by marclaporte** [comment]
@kroky what do you think?

**Comment by kroky** [comment]
Yes, +1 for bundling the vendor packages with the release tarballs.

**Comment by Shadow243** [comment]
It seams like the GITHUB_TOKEN provided by GitHub Actions has limited permissions by default. I'l review it and create a new one with white access permission.

**Comment by Shadow243** [comment]
@marclaporte @kroky @amessina Can we close this since the PR is already merged ?

**Comment by marclaporte** [comment]
> I'l review it and create a new one with white access permission

Is it done?

**Comment by Shadow243** [comment]
> > I'l review it and create a new one with white access permission
> 
> 
> 
> Is it done?

since the last release we have cypht.tar.gz which is added to the release and which contains the vendor folder already with the dependencies

https://github.com/cypht-org/cypht/pull/1142
